### PR TITLE
Update content and link on rejected declarations page list

### PR DESCRIPTION
**Pull request by russellthorn**
The story raised the issue of screen readers not reading the context of the content correctly. The knock on effect of this, was that content was injected into the change link with hidden text, along with the html markup for ordered list.

The fix was to replace the ordered list with plain English to improve context and readability for all users.

Unit tests have been run and all passed.

## Before
Screenshot 2020-11-16 at 15 44 16

## After
Screenshot 2020-11-16 at 15 43 44

### CODE REVIEW

**Review by MaciejRewera — COMMENTED** [review]
I'd rather do the same for all similar cases. As you can see there is quite a bunch of them. I'm not saying it should be you. If you do not have enough time, then maybe we could leave this branch, create a ticket for it, and then someone else can pick it up?

**Review by russellthorn — COMMENTED** [review]
Nicely done! 👍

**Comment by PauloPortugal** [comment]
I like the commit messages :)

File: conf/messages.en [hunk 1]
```diff
@@ -1298,34 +1298,34 @@ field.declaration.items.size = Total number of items
...
 field.declaration.borderTransport.meansOfTransportCrossingTheBorderType = Active means of transport type
 field.declaration.borderTransport.meansOfTransportCrossingTheBorderNationality = Active means of transport nationality
 field.declaration.borderTransport.modeCode = Mode of transport leaving the border
-field.declaration.parties.carrierDetails.details.eori = Carrier's EORI
-field.declaration.parties.carrierDetails.details.address.fullName = Carrier's name
-field.declaration.parties.carrierDetails.details.address.townOrCity = Carrier's town or city
-field.declaration.parties.carrierDetails.details.address.country = Carrier's country
-field.declaration.parties.carrierDetails.details.address.addressLine = Carrier's address line 1
-field.declaration.parties.carrierDetails.details.address.postCode = Carrier's postcode
+field.declaration.parties.carrierDetails.details.eori = Carrier’s EORI
```
Comment (PauloPortugal) [inline conf/messages.en:1309]: Ahhhhh!!!! I never thought I would see this one here. A GDS Standard! :+1:

**Review by PauloPortugal — APPROVED** [review]

File: conf/messages.en (hunk 1 above)
Comment (MaciejRewera) [reply conf/messages.en:1309]: Can't find it on my keyboard...

**Review by MaciejRewera — APPROVED** [review]

**Pull request merged**